app/actions/views/channel.js

```diff
@@ -607,13 +607,15 @@ export function loadChannelsForTeam(teamId, skipDispatch = false) {
             for (let i = 0; i <= MAX_RETRIES; i++) {
                 try {
                     console.log('Fetching channels attempt', teamId, (i + 1)); //eslint-disable-line no-console
-                    const [channels, channelMembers] = await Promise.all([ //eslint-disable-line no-await-in-loop
+                    const [channels, channelMembers, team] = await Promise.all([ //eslint-disable-line no-await-in-loop
                         Client4.getMyChannels(teamId, true),
                         Client4.getMyChannelMembers(teamId),
+                        Client4.getTeam(teamId),
```

Comment:
You should already have the team, why is this needed?

Reply:
Yep. Right! I forgot I could get the current team from the state. Thanks